configs/react.js

```diff
@@ -16,5 +16,31 @@ module.exports = {
 
     // require rel="noopener" or rel="noreferrer" with target="_blank" for security
     "react/jsx-no-target-blank": ["error", { allowReferrer: true }],
+
+    // deprecated mui/styled in our frontend code in favor of tss-react/mui
+    "@foxglove/no-restricted-imports": [
```

Comment:
```suggestion
    "@foxglove/no-mui-styled": [
```

Probably better to give this a more specific name if the list of rules is specific to mui styled.

Reply:
That custom rule is a passthrough of `no-restricted-imports` in eslint's built-in rules; it only becomes mui-specific when it's used within the `@foxglove/react` config with the mui params — and the config already has "react" in its name that makes it specific.
I don't think we need to rename `@foxglove/no-restricted-imports` here. That name tells us exactly what it is, how to use it and what we can use it for.